### DISCUSSION TITLE
v2.0.4

### DISCUSSION THREAD

**Discussion by aaronhelton** [dag-hammarskjold-library/dlx-rest#1056] [General]
## What's Changed

Updated dependencies (security fixes), which don't have direct user-facing implications:
* Bump cryptography from 36.0.2 to 39.0.1 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1049
* Bump werkzeug from 2.0.1 to 2.2.3 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/976

Updates with direct user-facing implications:
* 988/1002/1003 by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1052
  * Skips duplicate checking on authorities when only the indicator changed.
  * Improves searching so that headings with diacritics don't evaluate as identical to headings without diacritics (caused false positive duplicate detection)
  * Prevents headings with $g and headings without $g from evaluating as duplicate.
* Issue 871 by @Yalshire2004 in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1046
  * Adds a subsystem to manage field visibility and editability. We can now configure fields to be visible and/or editable. 
  * Additionally includes visual indicators when a field is not editable.

**Full Changelog**: https://github.com/dag-hammarskjold-library/dlx-rest/compare/v2.0.3-rc...v2.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/dag-hammarskjold-library/dlx-rest/releases/tag/v2.0.4'>v2.0.4</a>.</em>

**Comment by aaronhelton** [comment]
Deployed to QAT 2023-04-25

**Reply by viola-v** [reply]
Tested 2023-04-25 and reopened issue #1003
